datasets/ro/fiu_declarations/crawler.py

```diff
@@ -0,0 +1,35 @@
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def crawl(context: Context):
+    doc = context.fetch_html(context.data_url)
+    table = doc.find(".//table[@class='table table-bordered declarations']")
+    rows = table.xpath("./tr")
+    for row in rows:
+        first_td = row.xpath("./td[1]")
+        if first_td:
+            name = first_td[0].text_content().strip()
+
+            pep = context.make("Person")
+            pep.id = context.make_id(name)
+            pep.add("name", name)
+            pep.add("country", "ro")
+            pep.add("topics", "role.pep")
+            position = h.make_position(
+                context,
+                name="FIU Public Official",
```

Comment:
How about this instead? There's a lot of use of `FIU Official` in whitepapers but it's small enough to expand the acronym then I think it's really clear

```suggestion
                name="Financial Intelligence Unit Official",
```